app/views/mountain_view/styleguide/show.html.erb

```diff
@@ -44,10 +44,10 @@
   <% end %>
 <% else %>
   <div class="mv-hint">
-    <h2><span>Hint:</span>To see your component make sure you've created stubs:</h2>
+    <h2><%= t('mountain_view.show.hint_stub_creation_html') %></h2>
     <ul>
       <li>
-        You should write stub examples here:
+        <%= t('mountain_view.show.tip_stub_creation_html') %>:
```

Comment:
The `:` at the end of the line should go in the translation string (in `en.yml`)

Reply:
I can do that.  I had originally excluded the : as something we didn't want people modifying, but in my head I can go either way on that.  I'll shift it

Reply:
Thanks! I see it as more flexible if the colon is in the translation value. It could be that in some language the colon is not needed or doesn't fit. Or maybe somebody wants to just phrase it differently.